### PR TITLE
Enable the Turkish locale

### LINKED ISSUE #6828: Enable Turkish locale
As requested from localizers by email.

### PR DESCRIPTION
**Pull request by muffinresearch**
Fixes #6828

### CODE REVIEW

File: src/olympia/lib/settings_base.py [hunk 1]
```diff
@@ -192,6 +192,7 @@ def cors_endpoint_overrides(internal, public):
     'sl',  # Slovenian
     'sq',  # Albanian
     'sv-SE',  # Swedish
+    'tr', # Turkish
```
Comment (eviljeff) [inline src/olympia/lib/settings_base.py]: flake8 fail - only 1 space.

Comment (muffinresearch) [reply src/olympia/lib/settings_base.py]: potato

**Comment by muffinresearch** [comment]
@EnTeQuAk are there any other considerations re: adding a locale?

**Comment by EnTeQuAk** [comment]
It's supported by product-details and we have a few tests that theoretically make sure things work as expected. So, I'd say lgtm!

**Pull request merged**